charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -400,4 +400,11 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `backup.podSecurityContextOverride`      | Completely overwrites the contents of the backup pod's security context, ignoring the values provided for `runAsUser`, and `fsGroup`. | Not set |
 | `cronJob.apiVersion`                     | CronJob API version | 'batch/v1' |
 | `awsSecurityGroupPolicies.enabled`      | Enable the creation of SecurityGroupPolicy resources | `false` |
-| `awsSecurityGroupPolicies.policies` | Security Group Policy definitions. `awsSecurityGroupPolicies.enabled` must be `true`  | Not set |
\ No newline at end of file
+| `awsSecurityGroupPolicies.policies` | Security Group Policy definitions. `awsSecurityGroupPolicies.enabled` must be `true`  | Not set |
+
+### Unit Test
+
+| Parameter                      | Description                       | Default         |
+| ------------------------------ | --------------------------------- | --------------- |
+| `unittest.testFramework.image` | Image used to test the framework  | `bats/bats`     |
```

Comment:
```suggestion
### Helm Tests

| Parameter                      | Description                       | Default         |
| ------------------------------ | --------------------------------- | --------------- |
| `helmtest.bats.image` | Image used to test the framework  | `bats/bats`     |
```

I checked a bit the helm documentation and there best practices prefer [flat over nested values](https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values), but I see you point here of being able to configure more tests if needed.

It's just the term 'unittest`, which is heavily overloaded. Some people might feel the helm test, which deploy the chart and run tests against it are not unit tests. Personally I just fear that people confuse this with `helm unittest` which is also used in this chart.

I could not find a specific term being used in the [helm documentation](https://helm.sh/docs/topics/chart_tests/#helm) for these tests. As one uses [helm test](https://helm.sh/docs/helm/helm_test/#helm-test) to execute those I though about using `helmtest` as prefix.


I see where you `testFramework` values is coming from as the name of the init container is `test-framework`. I would simple name it `bats` as it's not really possible to use another test framework as we also do not allow to configure this:

```
      args:
        - |
          # copy bats to tools dir
          set -ex
          cp -R /opt/bats /tools/bats/
```

Reply:
Great suggestion and explanation, I fully agree! Reading through the helm documentation you provided I think your logic makes sense and will be more easily understood by other developers. I am terrible at naming terms and always love to hear from others in pull request! Appreciate you suggestion and will have an updated pull request soon! Hope you have an amazing Friday!